jcl-custom-rules/lib/README.md

```diff
@@ -0,0 +1,2 @@
+This directory should contain file sonar-jcl-plugin-1.2.0.1148.jar (the exact version number must match
```

Comment:
Suggestion: should contain *a* file named ... of *the* property ... in *the* file
That would also apply to the README in the cobol example [here](https://github.com/SonarSource/sonar-custom-rules-examples/blob/master/cobol-custom-rules/lib/README.md).